src/main/java/edu/ucsb/cs156/courses/documents/Update.java

```diff
@@ -0,0 +1,21 @@
+package edu.ucsb.cs156.courses.documents;
+
+import org.bson.types.ObjectId;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Document(collection = "updates")
+public class Update {
+    private ObjectId _id;
+    private String subject_area;
+    private String quarter_yyyyq;
+    private String last_update;
```

Comment:
This is probably better:

```suggestion
    private ZonedDateTime last_update;
```

See: Job.java under `entities` for examples (and the `import` that's needed).